### ISSUE TITLE
Move Travis jobs to travis-ci.com

### ISSUE THREAD

**Issue by nlohmann** [nlohmann/json#2938]
The Travis jobs are currently not triggered as the jobs are not yet moved to http://travis-ci.com.

**Comment by akbhuker** [comment]
hey, can I have this issue?
I'm quite new to this so I might need some guidance

**Comment by nlohmann** [comment]
Sure, give it a try. Though I have little knowledge of Travis myself and we only use it, because they offer some compiler versions nobody else do. I guess there are some documentation how to move projects, though I'm not sure whether this needs my credentials. Just try!

**Comment by akbhuker** [comment]
![Travis Jobs](https://user-images.githubusercontent.com/47684610/129856092-6e573a31-d7e7-48dd-a5e0-ce2e5fe9b1a3.JPG)

Hey @nlohmann I build jobs on travis-ci.com what to do now ?

**Comment by nlohmann** [comment]
I assume you forked the project and edited the YAML file for Travis? If so, please open a pull request.

**Comment by nlohmann** [comment]
@akbhuker FYI I just logged into travis-ci.com and could "migrate" my projects. Now I can see all previous builds there, but new builds are not triggered on new commits.

**Comment by akbhuker** [comment]
ok what should I do now?

**Comment by nlohmann** [comment]
Can you comment on https://github.com/nlohmann/json/pull/2940#pullrequestreview-732747252 ? If your change is more than converting YAML to JSON, then maybe I need to merge these changes to take effect. Otherwise, I'd need to contact the support of Travis.

**Comment by akbhuker** [comment]
I think merge these changes help you in to triggered build on new commits.

**Comment by akbhuker** [comment]
HI @nlohmann would you now able to trigger builds on new commits

**Comment by nlohmann** [comment]
@akbhuker I found the reason the projects did not build on travis-ci.com after the migration. I forgot to set this switch:

<img width="265" alt="image" src="https://user-images.githubusercontent.com/159488/130228532-20561608-9e7f-4b79-8479-0f1bcee4bcd1.png">

Now all there is left is to update the badge in the README. I will do this in a PR in a minute.

Sorry for the inconvenience - I did not know that I basically just had to press some buttons in the Travis UI.
